tests/test_packages/functional/package_test/__init__.py

```diff
@@ -0,0 +1,273 @@
+# coding=utf-8

```

Comment:
General comment on getting the environment variables.
The preferred way is to use os.environ.get('ENV_VAR', default_value).
This will not throw a KeyError exception if the ENV_VAR is not set and will return the default_value.